docs/operator/installing_server.md

```diff
@@ -200,429 +200,142 @@ becomes `RUCIO_CFG_DATABASE_DEFAULT`. All available environment variables are:
-__IdP nickname__ stands for your preferred IdP (e.g. 'wlcg'). The IdP
-specified under __admin_issuer__ will be contacted to get information about Rucio
-Users (SCIM) and to request tokens for the Rucio __root__ account.  The
-__expected_scope__ and __expected_audence__ parameters are optional and if not filled,
-the Rucio server will set them to `openid profile` and `rucio`
-respectively. The expected scopes and audiences have to be configured
-correspondingly on the side of your registered clients at your IdP (usually you
-can control accepted scopes and audiences for your clients via an IdP web
-interface).
-
-To finalize the process, one should assign the OIDC identities to the relevant
-Rucio __admin_account__ (e.g. 'root', 'ddmadmin'). This identity ID is
-composed of the Rucio Service IAM Account [A] subject claim and
-issuer url such as demonstrated below:
-
-```bash
-# Add the Rucio Service IAM Account ID as an OIDC identity
-rucio-admin identity add --account rucio_admin_account \
-  --type OIDC \
-  --id "SUB=b3127dc7-2be3-417b-9647-6bf61238ad01, \
-    ISS=https://wlcg.cloud.cnaf.infn.it/" \
-  --email "wlcg-doma-rucio@cern.ch"
 ```
 
-A second identity has to be added to the same __admin_account__ representing
-the __client_credentials__ flow of the Rucio application, i.e.  of the
-__Rucio Admin IAM Client__ [C2] from above. This identity consists of
-the __client_id__ of [C2] and the issuer (the token obtained via the
-client credentials flow using [C2] will contain in the __sub__ claim the
-__client_id__ of [C2] instead of Rucio Service IAM Account [A] __sub__ claim):
+#### Requiring extra Scopes
+If you want to add extra scope of ID token for authentication you can add it as required in server.
 
-```bash
-# Add the Rucio Admin IAM Client client_id as an OIDC identity
-rucio-admin identity add --account rucio_admin_account \
-  --type OIDC \
-  --id "SUB=5b5e5d37-926b-4b42-8a98-a0b4b28baf18, \
-    ISS=https://wlcg.cloud.cnaf.infn.it/" \
-  --email "wlcg-doma-rucio@cern.ch"
+Scopes can be extended if needed using the following configuration and corresponding claims existence for the scopes
+```cfg
+[oidc]
+id_token_extra_scopes= email, extra_scope
+id_token_extra_claims= email, claim_of_extra_scope
 ```
 
-Note: In case you can not/will not run any IAM -> Rucio user mapping tool in
-order to sync Rucio accounts with their IAM identities, you should assign the
-appropriate OIDC identity manually (as in the example above) to each Rucio
-account which is meant to use the OIDC authN/Z:
-
-```bash
-# Add an IAM User Account ID as an OIDC identity
-# (needs to be done for each user!)
-rucio-admin identity add --account rucio_user_account \
-  --type OIDC \
-  --id "SUB=5b5e5d37-926b-4b42-8a98-a0b4b28baf18, \
-    ISS=https://wlcg.cloud.cnaf.infn.it/" \
-  --email "wlcg-doma-rucio@cern.ch"
-```
+Only claim existence is check for now. If you need functionality for exact claim's value match please contact us we can add that feature too.
```

Comment:
Expanding this to add the feature request link to clarify where they should reach out

```suggestion
Only claim existence is checked for now. Please open a [feature request](https://github.com/rucio/rucio/issues/new?template=feature.yaml) if you would be interested in having this check expanded to take into account the claim value as well.
```